docs/src/guides/spelling.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Also provide an Earthly example how to play with spell checking.
You can just copy that code which you have already written with `check-spelling` and `check` target